pkg/controller/registry/resolver/resolver_test.go

```diff
@@ -141,10 +129,9 @@ func TestSolveOperators_MultipleChannels(t *testing.T) {
 	operators, err := satResolver.SolveOperators([]string{"olm"}, csvs, subs)
 	assert.NoError(t, err)
 	expected := OperatorSet{
-		"packageA.v1": genOperator("packageA.v1", "0.0.1", "packageA.v1", "packageA", "alpha", "community", "olm", nil, nil, nil, "", false),
 		"packageB.v1": genOperator("packageB.v1", "1.0.0", "", "packageB", "alpha", "community", "olm", nil, nil, nil, "", false),
 	}
-	assert.Equal(t, 2, len(operators))
+	assert.Len(t, operators, 1)
```

Comment:
discussed offline: this change confused me, but the old assertions were incorrect because packageA.v1 replaced itself